src/generators/genjs.ml

```diff
@@ -1418,6 +1418,16 @@ let generate_class_es6 ctx c =
 		end else
 			p
 	in
+
+	if not ctx.js_modern then
+		(try
+			let (_, args, pos) = Meta.get Meta.Expose c.cl_meta in
+			match args with
+			| [ EConst (String("default",_)), _ ] -> print ctx "export default ";
+			| [] -> print ctx "export ";
+			| _ -> abort "Invalid @:expose parameters" pos
```

Comment:
Not sure what to do in this case, since `@:export('something')` was valid until now and still has some use here.
Maybe this feature would be better with a `@:export` meta instead?

Reply:
I think `@:expose('CustomName') class Test{}` should be generated as `class Test {}; export {Test as CustomName}`